src/openrct2/ride/TrackPaint.h

```diff
@@ -28,6 +30,21 @@ extern const uint8_t edges_4x4[];
 
 extern const uint8_t track_map_1x4[][4];
 
+namespace TrackPaint::Flat
+{
+    // Lift and No lift sprites
```

Comment:
I think it should start with "no lift" and then continue with yes lift. Same for uninverted/inverted. The basic idea is to start with the simple/"undecorated" first.